guide/scss/_site-variables.scss

```diff
@@ -45,308 +43,42 @@ $sizes: map-merge(
 // Settings for the `<body>` element.
 
 $body-bg:                   #f5f5f5 !default;
-// $body-color:                $gray-900 !default;
 
 // Links
 //
 // Style anchor elements.
 
 $link-color:                $primary !default;
-// $link-decoration:           none !default;
-// $link-hover-color:          darken($link-color, 15%) !default;
 $link-hover-decoration:     none !default;
 
-// Paragraphs
-//
-// Style p element.
-
-//$paragraph-margin-bottom:   1rem !default;
-
 
 // Components
 //
 // Define common padding and border radius sizes and more.
 
-// $line-height-lg:              1.5 !default;
-// $line-height-sm:              1.5 !default;
-
-// $border-width:                1px !default;
-// $border-color:                $gray-300 !default;
-
-// $border-radius:               .25rem !default;
-// $border-radius-lg:            .3rem !default;
-// $border-radius-sm:            .2rem !default;
-
 $box-shadow-sm:               0 .125rem .25rem rgba($black, .075) !default;
 $box-shadow:                  0 .5rem 1rem rgba($black, .15) !default;
 $box-shadow-lg:               0 1rem 3rem rgba($black, .175) !default;
 
-// $component-active-color:      $white !default;
-// $component-active-bg:         theme-color("primary") !default;
-
-// $caret-width:                 .3em !default;
-
-// $transition-base:             all .2s ease-in-out !default;
-// $transition-fade:             opacity .15s linear !default;
-// $transition-collapse:         height .35s ease !default;
-
-
-// Fonts
-//
-// Font, line-height, and color for body text, headings, and more.
-
-// stylelint-disable value-keyword-case
-// $font-family-sans-serif:      "St. Jude Sans", "Open Sans", "Helvetica Neue", "Helvetica", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
-// $font-family-monospace:       SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace !default;
-// $font-family-base:            $font-family-sans-serif !default;
-// stylelint-enable value-keyword-case
-
-
-$font-size-base:              1rem !default; // Assumes the browser default, typically `16px`

```

Comment:
Was this intentionally deleted?

Reply:
Yes. Since our theme sets $font-size-base to 1.125rem(18px) based on the new style guide, I am assuming it would be better to have the guide represent the theme font size.